example/40-validatingwebhookconfiguration.yaml

```diff
@@ -22,4 +22,24 @@ webhooks:
   sideEffects: None
   clientConfig:
     url: "https://localhost:9443/webhooks/validate"
```

Comment:
There is no need to adapt anything. The gardener-apiserver is running on localhost and it can reach the webhook server running also on localhost. The resource for the newly introduced webhook is Secret and the kube-apiserver (if minikube is used) cannot by default reach the webhook on localhost. That's why `hack/dev-setup-admission-gcp.sh` is introduced.

Reply:
Hm, doesn't it make sense to also point to the `gardener-extension-admission-gcp` service for consistency reasons?

Reply:
I have no strong opinion, I now modified both to point to the service.

Reply:
Actually it wouldn't work with the service approach for local setup which is using minikube for example as the gardener-apiserver is not able to call the service. Probably something similar would work with the nodeless setup?
I am bringing it back to url for now.

Reply:
Yes makes sense, thank you.